src/Form/Field/ObjectChoiceList.php

```diff
@@ -34,33 +31,37 @@ public function __construct(
         $this->withWarning = $withWarning;
     }
 
-    public function getTypes()
+    public function getTypes(): string
     {
         return $this->types;
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
+     *
+     * @return array<mixed>
      */
-    public function getChoices()
+    public function getChoices(): array
     {
         $this->loadAll($this->types);
 
         return $this->choices;
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods